lib/plugins/expressionParser.js

```diff
@@ -163,6 +165,19 @@ function expressionApplier(context, typeInformation) {
 
         newAttribute.value = applyExpression(attribute.expression, context, typeInformation);
 
+        if (attribute.type === 'Integer') {
+            newAttribute.value = Number.parseInt(newAttribute.value);
+        }
+        else if (attribute.type === 'Float') {
+            newAttribute.value = Number.parseFloat(newAttribute.value);
+        }
+        else if (attribute.type === 'Boolean') {
+            newAttribute.value = (newAttribute.value === true);
+        }
+        else if (attribute.type === 'Null') {
```

Comment:
NGSIv2 uses "None" for this (see http://telefonicaid.github.io/fiware-orion/api/v2/stable/ section "Partial representations". Could make sense to change to align?

Reply:
Fixed in 4026d75714dd6ef51230bd01c8482c0a89e36923